### DISCUSSION TITLE
going back from "goto definition"

### DISCUSSION THREAD

**Discussion by skariel** [ki-editor/ki-editor#446] [Q&A] [answered]
say I press "g" then "d" and go to the definition of some symbol. How do I go back to the file/location I was before? In vim there is a jump list, pressing ctrl-o will ge me there. In ki the ctrl-o seems to only affect the seleciton, not the location or my position in the jump-list 

thanks!

**Comment by wongjiahau** [comment, marked as answer]
If `g d` brought you to a different file, you can use `{` to go back to the previous file.
